Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -18,6 +19,20 @@ class BillingFormTextFieldView: UIView {
 
     // MARK: - UI elements
 
+    private lazy var stackView: UIStackView = {
+        let view = UIStackView().disabledAutoresizingIntoConstraints()
+        view.axis = .vertical
+        view.spacing = 6
+        return view
+    }()
+
+    private lazy var headerStackView: UIStackView = {
+        let view = UIStackView().disabledAutoresizingIntoConstraints()
+        view.axis = .vertical
+        view.spacing = 6
+        return view
+    }()
+
```

Comment:
Given we're trying to add validation to the phone number, I am unsure why this is needed? I would expect new UI components if we were focussed on changing UI, but that is not what I thought we're doing?

Reply:
Every time there's an error for phone number, Sometimes, UI was not showing error view and sometime hide hint label.   Old UI was depending on constraints, I think, Stackviews is better for this solution.